library/Icingadb/Common/ObjectInspectionDetail.php

```diff
@@ -42,6 +42,11 @@ public function __construct(Model $object, array $apiResult)
         $this->joins = $apiResult['joins'];
     }
 
+    /**
+     * Render the object source location
+     *
+     * @return array|void
```

Comment:
`void` is only valid if there's not a single return statement, not even `return;` or `return null;`. So this should be `?array` instead.